ANTLR/src/Compiler/CodeGeneration/JavaScriptCodeGenerationVisitor.java

```diff
@@ -215,30 +223,23 @@ String GetFuncName(FunccallContext ctx){
      * @return The same number as in the source code.
      */
     @Override
-    public String visitValNumber(ValNumberContext ctx) {
+    public String visitExprNumber(ExprNumberContext ctx) {
         return ctx.NUMLITERAL().getText();
     }
 
-    /**
-     * Generates code for a boolean.
-     * @param ctx The tree node in question.
-     * @return The same boolean as in the source code.
-     * Therefore, the string, which is returned, can either be "{@code true}" or "{@code false}").
-     */
     @Override
-    public String visitValBoolean(ValBooleanContext ctx) {
+    public String visitExprBoolean(ExprBooleanContext ctx) {
         return ctx.BOOLLITERAL().getText();
     }
 
-
     @Override
-    public String visitValId(ValIdContext ctx) {
-        return ctx.ID().getText();
+    public String visitExprId(ExprIdContext ctx) {
+        return ctx.getText();
     }
```

Comment:
Det her ser mærkeligt ud. Det må du gerne forklare mig.

Reply:
Har rettet det til ctx.ID().getText() for læsbarhed, men vi kan sagtens lige snakke om det